default_configuration.yml

```diff
@@ -2,6 +2,10 @@ dataSources:
   - executor: FirebaseSource
   - executor: RedditCrawler
 processingStages:
+  - executor: Filterer
```

Comment:
Why are there two Filterers here?

Reply:
That's what we had before, the first one has no config so it does the default filter of removing finishing state. The error was the step to set the state to "in progress" was always before this filter.